docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,92 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend packages
+### The Scalable Pixel Streaming Frontend library
+The library is the part of the Scalable Pixel Streaming Frontend that consumes the Epic Games Pixel Streaming Frontend and UI Frontend. It includes all of the custom signalling server logic that Scalable Pixel Streaming signalling servers require to work. The library can be either obtained through [Github](https://github.com/ScalablePixelStreaming/Frontend) or [NPM](https://www.npmjs.com/package/@tensorworks/libspsfrontend). To make use of the library it must be initialised via HTML and Javascript. The library is written in TypeScript but configured to export as a UMD module and can be consumed by plain JavaScript and most JavaScript frameworks.  
+
+### The Scalable Pixel Streaming Frontend TypesScript example
+The TypeScript example is a simple HTML, CSS and TypeScript implementation of what a user could create to initialize the Scalable Pixel Streaming Frontend library. Its role is to instantiate the library's components and help start the Scalable Pixel Streaming connection to the signalling server.
```

Comment:
can we link to the example directory here please?

Reply:
Yes but it wont go anywhere until this stuff gets merged into main.

Reply:
Yes but be forewarned it will be broken until we merge this into main.